src/infinite-list/index.ts

```diff
@@ -1,8 +1,21 @@
-import { consume, tag, Tag } from '@storefront/core';
+import { provide, tag, Tag } from '@storefront/core';
 import List from '../list';
 
-@consume('infinite')
+@provide('infiniteList', (props) => props)
 @tag('gb-infinite-list', require('./index.html'), require('./index.css'))
 class InfiniteList extends List {}
 
+namespace InfiniteList {
+  export interface Props extends List.Props {
+    clickPrev: () => void;
```

Comment:
Add this param definition `event: MouseEvent & Tag.Event` (since I noticed it from documentation)